docs/README.md

```diff
@@ -41,6 +42,14 @@ In the product theme template, add the `condition-layout.product` block as a chi
   },
 ```
 
+Or the `condition-layout.binding` block, for example:
+```
+{
+  "store.product": {
+    "children": ["condition-layout.binding"]
+  },
+```
```

Comment:
Could you make this a valid JSON?
Removing the trailing comma and adding a new line with a closing curly brace should do it.

Reply:
I used the existing example as a template, but sure, I'll do it